samples/subsys/rtio/sensor_batch_processing/README.rst

```diff
@@ -0,0 +1,75 @@
+.. zephyr:code-sample:: sensor_batch_processing
+   :name: Sensor batch processing
+   :relevant-api: rtio
+
+   Implements sensor device using RTIO for asynchronuous data processing.
```

Comment:
a typo + changing wording to follow same style as other samples (https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/templates/sample.tmpl#L5-L6)

```suggestion
   Implement a sensor device using RTIO for asynchronous data processing.
```

Reply:
Fixed